docs/source/vision_tensors.rst

```diff
@@ -1,13 +1,13 @@
-.. _datapoints:
+.. _vision_tensors:
 
-Datapoints
-==========
+VisionTensors
+=============
 
-.. currentmodule:: torchvision.datapoints
+.. currentmodule:: torchvision.vision_tensors
 
-Datapoints are tensor subclasses which the :mod:`~torchvision.transforms.v2` v2 transforms use under the hood to
+VisionTensors are tensor subclasses which the :mod:`~torchvision.transforms.v2` v2 transforms use under the hood to
 dispatch their inputs to the appropriate lower-level kernels. Most users do not
-need to manipulate datapoints directly and can simply rely on dataset wrapping -
+need to manipulate vision_tensors directly and can simply rely on dataset wrapping -
```

Comment:
Occurrences like this "vision_tensors" one should probably become "VisionTensors". But this can be done in a follow-up PR that cleans the docs